From 89a2da7c8bae95cf9225015489736e2fc434f4d9 Mon Sep 17 00:00:00 2001
From: Jernej Skrabec <jernej.skrabec@siol.net>
Date: Sat, 2 Jan 2021 16:35:31 +0100
Subject: [PATCH] sunxi: Don't enable referenced regulators

This break certain devices which need appropriate power on sequence.
---
 drivers/allwinner/axp/common.c | 3 ---
 1 file changed, 3 deletions(-)

--- a/drivers/allwinner/axp/common.c
+++ b/drivers/allwinner/axp/common.c
@@ -112,9 +112,6 @@ static bool should_enable_regulator(const void *fdt,
 	if (is_node_disabled(fdt, node)) {
 		return false;
 	}
-	if (fdt_getprop(fdt, node, "phandle", NULL) != NULL) {
-		return true;
-	}
 	if (fdt_getprop(fdt, node, "regulator-always-on", NULL) != NULL) {
 		return true;
 	}
